cglib/src/main/java/net/sf/cglib/core/EmitUtils.java

```diff
@@ -388,6 +388,14 @@ public static void push_object(CodeEmitter e, Object obj) {
         }
     }
 
+    /**
+     * @deprecated use {@link #hash_code(CodeEmitter, Type, int, CustomizerRegistry)} instead
+     */
+    @Deprecated
+    public static void hash_code(CodeEmitter e, Type type, int multiplier, final Customizer customizer) {
```

Comment:
can you also add
 `public static void append_string(CodeEmitter, Type, ArrayDelimiters, Customizer) `
? ... that method was lost in https://github.com/cglib/cglib/commit/196da25819cd2080f87306c391611f1113e3391a too.